client/html/pages/panel/jobs/index.html

```diff
@@ -90,7 +90,7 @@
                           <br/>
                           <strong>{{companyName}}</strong>
                         </p>
-                        <div class="contain-lg">{{descricao}}</div>
+                        <!--div class="contain-lg">{{{descricao}}}</div-->
```

Comment:
Yep, we still don't have a way to limit how much text is displayed. So, if a company writes a long text, it will push other's vacancy below. So I thought it would be better not to display the description until the user clicks on it to detail in another page. Got it? :)